### ISSUE TITLE
move DNS records

### ISSUE THREAD

**Issue by afeld** [fecgov/fec-infrastructure#55]
Hello! @konklone just pointed out that there are still DNS records for FEC sites in [TTS's DNS repository](https://github.com/18F/dns). Given that the system is now owned and operated by @fecgov and that we shouldn't have DNS for Moderate systems in that repo (https://github.com/18F/dns/pull/277), I think it would be good to:

1. [x] Add the Terraform config here
1. [x] Point the nameservers for the domains to FEC's AWS account
1. [x] Confirm that the domains are pointing correctly and still working
1. [x] Clear out the old records from the TTS repo

Sound ok?

**Comment by rogeruiz** [comment]
Here's a write up about this on our 18F blog, https://18f.gsa.gov/2018/08/15/shared-infrastructure-as-code/ @patphongs

**Comment by rjayasekera** [comment]
beta.fec.gov and transition.fec.gov DNS records moved to FEC AWS and they are removed from 18F TTS records
https://github.com/18F/dns/pull/327
https://github.com/fecgov/fec-dns/commits/master

**Comment by dorothyyeager** [comment]
Our side of this is done; waiting on 18F to delete this.

**Comment by rjayasekera** [comment]
18F removed DNS entries from their side. 
https://github.com/18F/dns/pull/341
Closing this issue.
We probably need to open another issue to transfer https://api.open.fec.gov and https://api-stage.open.fec.gov.
We will discuss how to handle above 2 entires at the next meeting with cloud.gov
